scripts/.style.yapf

```diff
@@ -0,0 +1,17 @@
+[style]
+based_on_style = pep8
```

Comment:
What's this? Is this a published style?

Reply:
Yep, it's the most widely adopted code styling standards for Python code. This line is basically initializing all default formatter values to be what pep8 uses.

https://www.python.org/dev/peps/pep-0008/